core/src/main/java/org/infinispan/partitionhandling/PartitionHandling.java

```diff
@@ -0,0 +1,11 @@
+package org.infinispan.partitionhandling;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.1
+ */
+public enum PartitionHandling {
+   DENY_READ_WRITES, // If the partition does not have all owners for a given segment, both reads and writes are denied for all keys in that segment.
```

Comment:
Please use `/** */` style comment to document the behaviour in javadoc properly.